src/components/Transactions.js

```diff
@@ -15,34 +15,37 @@ const useStyles = makeStyles((theme) => ({
 
 export default function Transactions({ transactions, txCount }) {
   const classes = useStyles();
-  const rows = [
```

Comment:
The pre-fork version of this component did not support autopolling transactions (trader joe doesn't have this filter); the voltage-info tx table depended on data stored in a global context, and did this procedure of buffering/deduplicating to avoid redundant renders. I'm performing the equivalent of [that procedure](https://github.com/voltfinance/voltage-info/blob/f1e63728096419dd1c7bd98bc29639ce387941b7/src/components/TxnList/index.js) here.